### ISSUE TITLE
Profile dropdown menu gets cut off on smaller screens

### ISSUE THREAD

**Issue by crisner** [publiclab/plots2#6574]
### Please describe the problem (or idea)

> What happened just before the problem occurred? Or what problem could this idea solve?

The dropdown menu, from profile picture on the top right corner, gets cut off on smaller screens in mobile devices. On further investigation with the device mode in chrome dev tools, it looks like the menu begins to get cut off when the device height goes under 735px.
<p><span>
<img src="https://user-images.githubusercontent.com/29401459/67738988-cf9f1c80-fa36-11e9-91f1-9e34bb6e429a.png" alt="screenshot of the issue" width="220"  />

<img src="https://user-images.githubusercontent.com/29401459/67739748-80a6b680-fa39-11e9-9997-4e66609d3499.png" alt="Screenshot of dashboard(iPhone 5_SE)" width="220"  />
</span></p>

> What did you expect to see that you didn't?

I expected to be able to scroll down to access the rest of the menu

### Please show us where to look

https://publiclab.org/dashboard

### What's your PublicLab.org username?

> This can help us diagnose the issue: 

christie_reni

### Browser, version, and operating system

> Many bugs are related to these -- please help us track it down and reproduce what you're seeing!

Chrome, android
****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site. 

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

**Comment by jywarren** [comment]
Ah, very helpful, thanks! And, it's not possible to drag/scroll the screen
to access the menu content that went off the edge?

Thanks!

**Comment by crisner** [comment]
Yes. I can see the scrollbar moving when I scroll the screen but the menu seems to be fixed making the content that are off the edge inaccessible.

**Comment by susanreenesaa** [comment]
@jywarren I think this issue is as a result of the fixed top navigation bar. I have looked at pages https://publiclab.org/donate, https://publiclab.org/, https://publiclab.org/chat and https://publiclab.org/dashboard, they all have the same problem. bez their bar is fixed. this also occurs even on the desktop when the screen is minimized to mobile phone size.
However, for pages like https://github.com/publiclab/plots2/issues/6573 and https://github.com/publiclab/ which have no fixed navigation bar, the menus are scrollable.
so do we just remove fixed-top(fixed on top) from the navigation menu or?

**Comment by jywarren** [comment]
Ah, interesting. Is it possible we could set the CSS for some element to
"overflow: scroll" to resolve this?

**Comment by susanreenesaa** [comment]
let me try it out.

**Comment by susanreenesaa** [comment]
I created an FTO issue out of this Feel free to review it. Thanks, everyone
